x-pack/plugins/secops/public/components/drag_and_drop/droppable_wrapper.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { Droppable } from 'react-beautiful-dnd';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+import { ThemeProvider } from 'styled-components';
+
+import * as euiVars from '@elastic/eui/dist/eui_theme_k6_light.json';
+
+interface Props {
+  droppableId: string;
+}
+
+const ReactDndDropTarget = styled.div<{ isDraggingOver: boolean }>`
+  transition: background-color 0.7s ease;
+  background-color: ${props =>
+    props.isDraggingOver ? '#D9D9D9' : props.theme.eui.euiColorEmptyShade};
```

Comment:
nit: prettier is missing this property but typically prettier linter would ask for you to make this `#d9d9d9`. So I would suggest just making this lowercase now.